cgi-bin/LJ/Talk.pm

```diff
@@ -1474,6 +1473,28 @@ sub talkform {
 
     my $screening = LJ::Talk::screening_level( $journalu, $opts->{ditemid} >> 8 ) // '';
 
+    # pre-calculate some abilities and add them to $remote, so we don't have to do it
+    # in the template
+    if ($remote) {
+        $remote->{can_manage_community} =
```

Comment:
I'm not comfortable adding fields to an LJ::User here like this. Is there any way we can just pass this as a separate hashref, some 'remote_icon_opts' or something idk?

Reply:
That might be possible! The original code made a new `$remote` object that consisted of these bits and a handful of properties from the original remote, but not all of them.